--- rpm/librpm4_stubs.c.orig	2014-12-16 14:22:37.202351015 +0000
+++ rpm/librpm4_stubs.c	2014-12-16 14:28:49.173378156 +0000
@@ -218,12 +218,12 @@
   tl = append(hd,tl);
 
   k = caml_copy_string("Suggests");
-  v = get_deps(h,RPMTAG_SUGGESTSNAME);
+  v = get_deps(h,RPMTAG_OLDSUGGESTSNAME);
   hd = tuple(k,list_variant_D(v));
   tl = append(hd,tl);
 
   k = caml_copy_string("Enhances");
-  v = get_deps(h,RPMTAG_ENHANCESNAME);
+  v = get_deps(h,RPMTAG_OLDENHANCESNAME);
   hd = tuple(k,list_variant_D(v));
   tl = append(hd,tl);
 
